functions/twist_angle.h

```diff
@@ -24,4 +24,23 @@ class TwistAngleSVF {
 template<int N = 2, int OFFSET = 0>
 using TwistAngle = SingleValueAdapter<TwistAngleSVF<N, OFFSET>>;
 
+// Usage: TwistAcceleration<MAX>
+// Returns 0-32768 based on acceleration of twist in one direction
+// MAX : Maximum acceleration needed to return 32768
+// returned value: FUNCTION, same for all leds
+
+template<class MAX>
+class TwistAccelerationSVF {
+ public:
+  void run(BladeBase* blade) { max_.run(blade); }
+  int calculate(BladeBase* blade) {
+   float v = (fabs(fusor.twist_accel()) * 32768 / 360) / max_.calculate(blade);
```

Comment:
I still think it's super weird to have a fabs() in here.
Might make more sense to multiply it by -1 if fusor.gyro().z < 0 ?
(This would make positive values mean that it is speeding up, regardless of whether it is twisting to the left or to the right.)

Reply:
Updated